stream.go

```diff
@@ -147,7 +145,7 @@ func (s *Stream) MeasurementLatest(p Params) (<-chan *measurement.Result, error)
         err := h.Emit("atlas_subscribe", subscribe)
         if err != nil {
             r := &measurement.Result{ParseError: fmt.Errorf("h.Emit(atlas_subscribe): %s", err.Error())}
-            ch <- r
+            trySend(ch, r)
             c.Close()
             close(ch)
```

Comment:
I think this close is redundant, since the channel is closed by the onDisconnection method called by c.Close()

Reply:
Nice catch Greg; let's land this change on our internal fork so we can test with it.